src/Rules/Documentation.hs

```diff
@@ -23,12 +23,16 @@ documentationRules = do
     buildPdfDocumentation
     buildDocumentationArchives
     buildManPage
+    "*//docs//gen_contents_index" %> \out -> copyFile "libraries/gen_contents_index" out
+    "*//prologue.txt" %> \out -> copyFile "libraries/gen_contents_index" out
```

Comment:
I'm confused by the file patterns above: why not `//docs/gen_contents_index` and `//prologue.txt`?

Reply:
Also, not a big deal, but there is a slight simplification: `\out -> copyFile "libraries/gen_contents_index" out` can be replaced with `copyFile "libraries/gen_contents_index"`.

Reply:
Fixed, thanks!